Source/ASTextNode2.mm

```diff
@@ -209,6 +209,15 @@ - (BOOL)supportsLayerBacking
   return YES;
 }
 
+- (NSString *)defaultAccessibilityLabel {
```

Comment:
Please add a newline here and the next one too

Reply:
Can you also add the `defaultAccessibilityLabel` to `ASTextNode` please.

Reply:
Done, added the default a11y stuff to ASTextNode.